examples/stable_diffusion_xl/tools/model_conversion/convert_diffusers_to_original_sdxl.py

```diff
@@ -0,0 +1,349 @@
+# Script for converting a HF Diffusers saved pipeline to a Stable Diffusion checkpoint.
```

Comment:
Add usage instructions in README
https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_xl/tools/model_conversion/README.md